src/__tests__/role-helpers.js

```diff
@@ -132,15 +131,6 @@ test('getRoles returns expected roles for various dom nodes', () => {
   })
 })
 
-test('logRoles logs expected roles for various dom nodes', () => {
```

Comment:
To test calls to `console.log` you could do something like this:

```javascript
jest.spyOn(console, 'log').mockImplementationOnce(() => {})

const {section} = setup()
logRoles(section)

expect(console.log).toHaveBeenCalledTimes(1)
expect(console.log.mock.calls[0][0]).toMatchSnapshot()

console.log.mockRestore()
```

Reply:
Nice! Looks like you figured it out without my help. Two things:

1. Let's move that `addSnapshotSerializer` stuff to the `setup-env.js` file.
2. Make sure to call `mockRestore()`

Reply:
Handled [5fefa1](https://github.com/testing-library/dom-testing-library/pull/312/commits/5fefa10d167518fa396538a09ef428b81f2ce45d)